src/index.js

```diff
@@ -0,0 +1,59 @@
+import './style.css';
+
+const tasksArr = [
+  {
+    description: 'Get out of bed',
+    completed: true,
+    index: 44,
+  },
+  {
+    description: 'Go to the gym',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Have breakfast',
+    completed: false,
+    index: 10,
+  },
+];
+
+const ul = document.querySelector('.list-container');
+
+const sortArr = () => {
+  tasksArr.sort((a, b) => a.index - b.index);
+};
+const generateList = () => {
+  for (let i = 0; i < tasksArr.length; i += 1) {
+    const li = document.createElement('li');
+    const check = document.createElement('input');
+    const lbl = document.createElement('label');
+    check.type = 'checkbox';
+    check.name = tasksArr[i].index.toString();
+    ul.append(li);
+    li.append(lbl);
+    lbl.append(check);
+    lbl.innerHTML += tasksArr[i].description;
+  }
```

Comment:
- [ ] Nice work implementing the list of tasks :clap: However your design is missing the left icon that help to remove a task which is located on the right side of each task
### Your design :point_down: 
![Screenshot from 2023-08-17 20-24-23](https://github.com/lucy-sees/CheckLister/assets/91142432/1fbb7a65-1756-4d04-9103-331bd95e2e96)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-08-17 20-23-52](https://github.com/lucy-sees/CheckLister/assets/91142432/8a62664c-d078-4ffc-9222-f99e1d7b7515)